README.md

```diff
@@ -1,2 +1,112 @@
 # Open Data for Tax Justice Pipelines and Processors
 
+The purpose of this pipeline is to collect and normalize all CRD IV data into a database.
+
+Currently we expect the data source to be PDF tables. Please open an issue to request support for other formats.
+
+## How to contribute a new dataset
+
+1. Check if the entity (e.g. bank, company) already has [a folder in our pipeline](https://github.com/okfn/datapackage_pipelines_od4tj/tree/master/datapackage_pipelines_od4tj/pipelines/crd-iv).
```

Comment:
Let's make it simpler:
- Download the file (but write down the download link)
- Open the file in tabula (locally installed or in tabula.openknowledge.io
- your stage 5 & 6 
- Email us (or open an issue) with the following information:
  - Download link
  - Company
  - Year
  - Dimension
  - Column mapping
  - Currency & units (e.g. millions)